docs/sources/mimir/_index.md

```diff
@@ -1,29 +1,58 @@
 ---
-description: Technical documentation for Grafana Mimir
-keywords:
-  - Grafana Mimir
-  - Grafana metrics
-  - time series database
-  - TSDB
-  - Prometheus storage
-  - Prometheus remote write
-  - metrics storage
-  - metrics datastore
-  - observability
-menuTitle: Grafana Mimir
-title: Grafana Mimir documentation
+description: Learn about the key benefits and features of Grafana Mimir.
+labels:
+  products:
+    - OSS
+title: Grafana Mimir
 weight: 1
+hero:
+  title: Grafana Mimir
+  level: 1
+  image: /media/docs/mimir/GrafanaLogo_Mimir_icon.png
+  width: 100
+  height: 100
+  description: Grafana Mimir is an open source software that provides horizontally scalable, highly available, multi-tenant, long-term storage for Prometheus.
+cards:
+  title_class: pt-0 lh-1
+  items:
+    - title: Get started
+      href: ./get-started/
+      description: Get started with Grafana Mimir
+      height: 24
+    - title: Configurations
+      href: ./configure/about-configurations/
+      description: Learn about configuration options for Grafana Mimir
+      height: 24
+    - title: Send metric data
+      href: ./send/
+      description: Configure your data source to write data to Grafana Mimir
+      height: 24
+    - title: Manage
+      href: ./manage/
+      description: Explore your options when deploying, configuring and managing Mimir
+    - title: Query metric labels
+      href: ./query/
+      description: Query metric labels with HTTP API endpoints.
+      height: 24
+    - title: Visualize data
+      href: ./visualize/
+      description: Use Grafana Explore and PromQL to visualize data in Grafana
+      height: 24
 ---
 
-# Grafana Mimir documentation
+{{< docs/hero-simple key="hero" >}}
 
-![Grafana Mimir](mimir-logo.png)
+---
+
+## Overview
+
+Experience the capabilities of Grafana Mimir for scalable, cost-effective, and reliable long-term storage of your Prometheus metrics.
```

Comment:
this leaves out things and generally looks less impressive to me - was it discussed with PM?

Reply:
@krajorama  This was essentially a training task for me to learn how to do the landing pages, with the intent of updating the descriptive information afterward.  I was delayed in handling another issue, but I intend to update the descriptions this week and this will be done in conjunction with the dev team/PM.  Thanks for the input though, I'll certainly use it as I update the content.  If you've any other suggestions, I'll gladly take them.

Reply:
Coming back to this, I believe @zhehao-grafana has approved the landing page changes or at least the plan to. 

@zhehao-grafana would you like to take a look at the changes in this PR and confirm you are happy with them?

Reply:
@zhehao-grafana  @krajorama  Alternate possibilities for the tiles (these are taken from the web report that shows the Mimir pages most accessed by users):
1 Grafana Mimir configuration parameters
2 Get started with Grafana Mimir (already have this one)
3 Grafana Mimir HTTP API
4. Get started with Grafana Mimir using the Helm chart
5. Grafana Mimir architecture
6. Grafana Mimirtool
7. Grafana Mimir runbooks
8. Set up Mimir

If any of these would be preferred over what's there now, I can change out the existing tiles and update the text with your help and contributions.

Reply:
@krajorama  The main objective in the change to the new landing page layout is to help users find information, so the tiles are meant to be a navigational aid to the information they're looking for (which will save them time searching for it).  In the old format, the landing page had more marketing-type information, which doesn't work when someone is coming to these pages to find out how to do something (and sometimes needs to find it quickly!)